### PR TITLE
DIMM: Report expiration date for DIMM measurements.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: sal_interfaces/DIMM/DIMM_Events.xml [hunk 1]
```diff
@@ -50,6 +50,13 @@
       <Units>second</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>expiresAt</EFDB_Name>
```
Comment (timj) [inline sal_interfaces/DIMM/DIMM_Events.xml:54]: `validUntil` is another option for you given the Description string you used below.

File: sal_interfaces/DIMM/DIMM_Events.xml [hunk 2]
```diff
@@ -50,6 +50,20 @@
...
+      <Description>Timestamp until when the measurement is deemed valid.</Description>
```
Comment (patrickingraham) [inline sal_interfaces/DIMM/DIMM_Events.xml:55]: do you want to add that it's in unix_tai ? might save headaches for people later.

File: sal_interfaces/DIMM/DIMM_Events.xml [hunk 3]
```diff
@@ -50,6 +50,20 @@
...
+      <IDL_Type>float</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>expiresIn</EFDB_Name>
+      <Description>How long the measurement is deemed valid after being published.</Description>
```
Comment (patrickingraham) [inline sal_interfaces/DIMM/DIMM_Events.xml:62]: Consider:
Duration the measurement is deemed valid since publication.

**Review by patrickingraham — APPROVED** [review]
Made two suggestions.

**Pull request merged**